content/02.main-text.md

```diff
@@ -317,6 +321,34 @@ While Manubot does require technical expertise, the benefits are manyfold.
 Specifically, Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
 These attributes empower integrating Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
 
+## Future Manubot Enhancements
+
+Manubot is still under active development, and we envision major changes in its design and dependencies going forward.
+Currently, manuscript repositories must contain a large number of files that do not directly contain manuscript content.
+While this enables a high-degree of customization, it also increases complexity.
+Therefore, we are investigating whether configuration files with sensible defaults could enable bare-bones repositories that contain manuscript content and little else.
+
+In addition, to simpler usage, we're also looking into simplified setup.
+Presently, setup is complex because users must do advanced command-line operations to clone the `manubot/rootstock` repository and configure Travis CI.
```

Comment:
This is the first time `manubot/rootstock` appears in the text.  If we add a "Getting Started with Manubot" section earlier, it will be okay.

Reply:
Okay, let's remember to do this next when addressing https://github.com/greenelab/meta-review/issues/155 and adding the "Getting Started" section